src/ui-lib/sass/05-components/03-organisms/00-page-structure/_01-page-footer.scss

```diff
@@ -26,21 +25,23 @@ body > footer {
   }
 
   svg {
-    display: block;
+    display: inline-block;
+    margin: 0 auto;
     width: auto;
```

Comment:
I think most of these properties are intended for the Buildit logo, not the icons.

Maybe change the selector to `.grav-c-logo`. That way, you don't need to undo all that stuff for the icons further down.